torch/nn/utils/parametrize.py

```diff
@@ -66,17 +66,20 @@ class ParametrizationList(ModuleList):
     Args:
         modules (iterable): an iterable of modules representing the parametrizations
         original (Parameter or Tensor): parameter or buffer that is parametrized
+        allow_resize (bool): a boolean flag that denotes whether the parametrized tensor can be resized
```

Comment:
I think we want to be a little more ominous here. Warning that the current Module might not work any-more and users enable this flag at their own risk.